attest/ake/Cargo.toml

```
@@ -20,13 +20,16 @@ sgx-sim = [
 [dependencies]
 mc-attest-core = { path = "../../attest/core", default-features = false }
 mc-attest-verifier = { path = "../../attest/verifier", default-features = false }
+mc-attest-verifier-types = { path = "../../attest/verifier/types", default-features = false }
 mc-crypto-keys = { path = "../../crypto/keys", default-features = false }
 mc-crypto-noise = { path = "../../crypto/noise", default-features = false }
 
 aead = "0.5"
+der = "0.7.7"
```

:wrench: Looks like elsewhere it's 0.7.8
```suggestion
der = "0.7.8"
```